pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -782,6 +765,7 @@ func TestRecentlyTriedPodsGoBack(t *testing.T) {
 		}
 		q.Add(&p)
 	}
+	c.Step(time.Microsecond)
```

Comment:
why is this necessary?

Reply:
In this case, it's used to make the not-tried Pods' timestamp a bit advanced than the tried Pod (pod-0). Otherwise, even pod-0 is tried once (in `q.Pop()` where its timestamp gets updated), its timestamp holds the same as the other not-tried ones, so in QueueSort, pod-0 would be very likely not popped as expected (we expect it's the last one to be popped).